File: src/main/java/net/gamershub/kitpvp/cosmetics/impl/ShriekKillEffect.java
package net.gamershub.kitpvp.cosmetics.impl;

import net.gamershub.kitpvp.cosmetics.KillEffect;
import org.bukkit.Location;
import org.bukkit.Material;
import org.bukkit.Particle;

public class ShriekKillEffect extends KillEffect {
    public ShriekKillEffect() {
        super("shriek", "Shriek", 1, Material.SCULK_SHRIEKER);
    }

    @Override
    public void playEffect(Location location) {
        location.getWorld().spawnParticle(Particle.SHRIEK, location, 10, 0, 0, 0, 0, 1);
    }
}

File: src/main/kotlin/io/github/chaosdave34/kitpvp/cosmetics/impl/ShriekKillEffect.kt
package io.github.chaosdave34.kitpvp.cosmetics.impl

import io.github.chaosdave34.kitpvp.cosmetics.KillEffect
import org.bukkit.Location
import org.bukkit.Material
import org.bukkit.Particle

class ShriekKillEffect : KillEffect("shriek", "Shriek", 2, Material.SCULK_SHRIEKER) {
    override fun playEffect(location: Location) {
        location.world.spawnParticle(Particle.SHRIEK, location, 10, 0.0, 0.0, 0.0, 0.0, 1)
    }
}